liquid.py

```diff
@@ -237,17 +260,60 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def collection_exists(name):
+    """Returns true if the collection with the given name was declared in the ini file."""
+
+    names = dict(config.collections).keys()
+    return reduce(lambda exists, curr_name: exists or curr_name.lower() == name.lower(), names, False)
+
+
+def initcollection(name):
+    """Initialize collection with given name.
+    
+    Create the snoop database, create the search index, run dispatcher, add collection
+    to search.
+    
+    :param name: the collection name
+    """
+    if not collection_exists(name):
+        raise RuntimeError('Collection %s does not exist in the liquid.ini file.', name)
+
+    container = f'snoop-{name}-api'
+
+    try:
+        shell_cmd(container, './manage.py', 'initcollection')
+    except CalledProcessError as e:
+        if not re.search(f'index \[{name}/[^\]]+] already exists', e.output.decode()):
+            log.error(e.output.decode())
+            raise
+        log.info(f'Index {name} already exists.')
+
+    try:
+        snoop_url = first(nomad.agent_members())['Addr'] + f':8765/{name}/collection/json'
+        shell_cmd('hoover-search', './manage.py', 'addcollection', name, '--index',
+                  name.lower(), snoop_url, '--public')
```

Comment:
Don't use `.lower()` here

Reply:
I need to, because the collection name might contain upper case letters.